packages/api-client/README.md

```diff
@@ -214,6 +215,23 @@ fromCatalog()
     .lang('fr'); // Force a language
 ```
 
+# Versioning
+
+The client must follow [Opendatasoft's Explore API](https://help.opendatasoft.com/apis/ods-explore-v2/) evolution.
```

Comment:
```suggestion
The client will follow [Opendatasoft's Explore API](https://help.opendatasoft.com/apis/ods-explore-v2/) evolution and versions.
```
must (we described an intent in a spec) -> will (we're describing what we're doing for real now)